parity/cli/subcommands.rs

```diff
@@ -0,0 +1,242 @@
+)]
+pub struct ExportState {
+    #[structopt(long = "no-storage", help = "Don't export account storage.")]
+    pub no_storage: bool,
+
+    #[structopt(long = "no-code", help = "Don't export account code.")]
+    pub no_code: bool,
+
+    #[structopt(
+        long = "max-balance",
+        name = "MAX_WEI",
+        help = "Don't export accounts with balance greater than specified."
+    )]
+    pub max_balance: Option<String>,
+
+    #[structopt(
+        long = "min-balance",
+        name = "MIN_WEI",
+        help = "Don't export accounts with balance less than specified."
+    )]
+    pub min_balance: Option<String>,
+
+    #[structopt(
+        default_value = "latest",
+        long,
+        name = "BLOCK",
+        help = "Take a snapshot at the given block, which may be an index, hash, or latest. Note that taking snapshots at non-recent blocks will only work with --pruning archive"
+    )]
+    pub at: String,
+
+    #[structopt(
+        long,
+        name = "FORMAT",
+        help = "Export in a given format. FORMAT must be either 'hex' or 'binary'. [default: binary]"
+    )]
+    pub format: Option<String>,
+
+    #[structopt(long = "file", name = "FILE", help = "Path to the exported file")]
+    pub file: Option<String>,
+}
+
+#[derive(StructOpt, Deserialize, Debug, Clone)]
+#[structopt(about = "Manage Signer")]
+pub enum Signer {
+    #[structopt(
+        about = "Generate a new signer-authentication token for the given --chain (default: mainnet)"
+    )]
+    NewToken,
+    #[structopt(
+        about = "List the signer-authentication tokens from given --chain (default: mainnet)"
+    )]
+    List,
+    #[structopt(about = "Sign")]
+    Sign {
+        #[structopt(name = "ID")]
+        id: Option<usize>,
+    },
+    #[structopt(about = "Reject")]
+    Reject {
+        #[structopt(name = "ID")]
+        id: Option<usize>,
+    },
+}
+
+#[derive(StructOpt, Deserialize, Debug, Clone)]
+#[structopt(about = "Make a snapshot of the database of the given chain(default: mainnet)")]
+pub struct Snapshots {
+    #[structopt(
+        default_value = "latest",
+        name = "BLOCK",
+        help = "Take a snapshot at the given block, which may be an index, hash, or latest. Note that taking snapshots at non-recent blocks will only work with --pruning archive"
+    )]
+    pub at: String,
+    // FIXME: check if the default is correct
+    #[structopt(name = "FILE", help = "Path to the file to export to")]
+    pub file: Option<String>,
+}
+
+#[derive(StructOpt, Deserialize, Debug, Clone)]
+#[structopt(
+    about = "Restore the databse of the given chain (default: mainnet) from a snapshot file"
+)]
+pub struct Restore {
+    #[structopt(name = "FILE", help = "Path to the file to restore from")]
+    pub file: Option<String>,
+}
+
+#[derive(StructOpt, Deserialize, Debug, Clone)]
+#[structopt(about = "Tools")]
+pub enum Tools {
```

Comment:
I think this command can safely be marked as deprecated, @vorot93 wdyt?

Reply:
I think it can be removed even.

Reply:
Okay, removing this also